client/my-sites/exporter/exporter.jsx

```diff
@@ -36,13 +36,15 @@ export default React.createClass( {
 	render: function() {
 		const { setPostType, startExport } = this.props;
 		const { postType, shouldShowProgress } = this.props;
+		const siteId = this.props.site.ID;
+		const onClickExportAll = () => startExport( siteId );
```

Comment:
Do we need the `onClick` in this name? It seems sort of like `exportAll` is the action and that it's a little redundant below to have `onClick={ onClickExportAll }`. I don't feel super strongly about it.